wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -732,10 +732,11 @@ object Wallet extends WalletLogger {
   def initialize(wallet: Wallet, bip39PasswordOpt: Option[String])(implicit
       walletAppConfig: WalletAppConfig,
       ec: ExecutionContext): Future[Wallet] = {
+    val password = walletAppConfig.aesPassword
```

Comment:
So why allow `bip39PasswordOpt` as a parameter, but not `aesPassword`? It seems we should have this precedent

1. The parameter passed in (think a user text box in GUI, or a command line parameter)
2. Read from the config file like you are doing here

I think this would follow the typesafe config resolution precedent as documented [here](https://github.com/lightbend/config#standard-behavior)


We should eventually  `bip39Password` to be in the wallet config file too.

Thinking about this further, we probably need to get issues filed for changing the `aesPassword` (easy, i think) and then `bip39Password` (complex and involves creating a new wallet, i think?)